src/main/java/mate/academy/springboot/swagger/dto/mapper/ProductMapper.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.springboot.swagger.dto.mapper;
+
+import mate.academy.springboot.swagger.dto.request.ProductRequestDto;
+import mate.academy.springboot.swagger.dto.response.ProductResponseDto;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductMapper {
+
+    public Product toModel(ProductRequestDto requestDto) {
+        Product product = new Product();
+        product.setTitle(requestDto.getTitle());
+        product.setPrice(requestDto.getPrice());
+        return product;
+    }
+
+    public ProductResponseDto responseDto(Product product) {

```

Comment:
Consider renaming the method 'responseDto' to 'toResponseDto' for better clarity.